cmd/minikube/cmd/start.go

```diff
@@ -148,6 +150,7 @@ func initMinikubeFlags() {
 	startCmd.Flags().String(networkPlugin, "", "The name of the network plugin.")
 	startCmd.Flags().Bool(enableDefaultCNI, false, "Enable the default CNI plugin (/etc/cni/net.d/k8s.conf). Used in conjunction with \"--network-plugin=cni\".")
 	startCmd.Flags().Bool(waitUntilHealthy, true, "Wait until Kubernetes core services are healthy before exiting.")
+	startCmd.Flags().Duration(waitTimeout, 3*time.Minute, "max time to wait for Kubernetes core services to be healthy.")
```

Comment:
This default seems quite short for certain environments: Previously, it was 5-minutes per pod. 5 minutes overall perhaps?

Reply:
I should had added Per Kubernetes service... ( this is not for all) and it used to be 5 min per componenent